app/models/validations/setup_validations.rb

```diff
@@ -77,7 +77,7 @@ def validate_scheme(record)
       record.errors.add :scheme_id, :incomplete, message: I18n.t("validations.lettings.setup.scheme.incomplete")
     end
 
-    unless record.scheme&.locations&.confirmed&.any?
+    if record.scheme&.locations.present? && !record.scheme.locations.confirmed.any?
       record.errors.add :scheme_id, :no_completed_locations, message: I18n.t("validations.lettings.setup.scheme.no_completed_locations")
     end
```

Comment:
Alternative: This triggered a lot of tests to fail as they tested schemes without creating completed locations. Perhaps we could update the factory for a scheme? But that might affect tests that purposely do not include a location on the scheme.